### PR TITLE
Campaign run cleanup

### LINKED ISSUE #5961: Create campaign runs in a script
Create campaign run nodes for all open campaigns (published or unpublished)

### PR DESCRIPTION
**Pull request by chloealee**
#### What does this PR do?

Creates campaign run nodes for all active campaigns (published or unpublished).  
#### How should this be manually tested?

When ssh'd in your vagrant environment, run `drush updb -y` in `dosomething@dev:/var/www/dev.dosomething.org/html$`

Note: this has not yet been tested in staging/Thor
#### What are the relevant tickets?

Fixes #5961

### CODE REVIEW

File: lib/modules/dosomething/dosomething_campaign_run/dosomething_campaign_run.install [hunk 1]
```diff
@@ -20,3 +20,54 @@ function dosomething_campaign_run_update_7002(&$sanebox) {
...
+                                GROUP BY t.entity_id;");
+
+  foreach ($active_campaigns as $active_campaign) {
+    // For each $active_campaign, create a new node (campaign run) with title of campaign + current year.
+    $campaign_run_node = new stdClass();
+    $campaign_run_node->type = 'campaign_run';
+    $campaign_run_node->language = $active_campaign->src_lang;
+    $created_dates = explode(',', $active_campaign->created_dates);
+
+    //TODO: change the title below per Fantini's guidelines
```
Comment (angaither) [inline lib/modules/dosomething/dosomething_campaign_run/dosomething_campaign_run.install]: you can remove this line now :)

File: scripts/campaign-run-cleanup.php [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+/*
+ * Script to create campaign runs for all open campaigns.
+ */
+
+// get all campaigns with status active
+$active_campaigns = db_query('SELECT n.nid, n.title, n.created
+FROM node n
+INNER JOIN field_data_field_campaign_status s on s.entity_id = n.nid
+WHERE field_campaign_status_value = 'active';');
+
```
Comment (angaither) [inline scripts/campaign-run-cleanup.php]: you can delete this file too!

**Comment by chloealee** [comment]
@angaither removed the comment and extra script file!

File: lib/modules/dosomething/dosomething_campaign_run/dosomething_campaign_run.install [hunk 3]
```diff
@@ -1,4 +1,4 @@
-<?php
+  <?php
```
Comment (angaither) [inline lib/modules/dosomething/dosomething_campaign_run/dosomething_campaign_run.install]: extra space!

**Comment by chloealee** [comment]
@angaither oops thanks for catching! Removed :)

**Comment by angaither** [comment]
sweeeeet, let's merge and test this!

**Pull request merged**